features/lorawan/lorastack/mac/LoRaMac.cpp

```diff
@@ -1348,6 +1350,13 @@ lorawan_status_t LoRaMac::prepare_join(const lorawan_connect_t *params, bool is_
             // Reset variable JoinRequestTrials
             _params.join_request_trial_counter = 0;
         } else {
+            if ((params->connection_u.abp.dev_addr == 0)    ||
+                (params->connection_u.abp.nwk_id == 0)      ||
+                (params->connection_u.abp.nwk_skey == NULL) ||
+                (params->connection_u.abp.app_skey == NULL)) {
+                return LORAWAN_STATUS_PARAMETER_INVALID;
```

Comment:
Change these to the same style as aboce done for OTAA checks. (|| operator at the beginning of the line.)